fossunited/foss_hackathon/doctype/foss_hackathon_participant/foss_hackathon_participant.py

```diff
@@ -57,6 +99,8 @@ def handle_localhost_request(self):
             return
 
         if (self.localhost == prev_doc.localhost) and self.localhost_request_status == "Rejected":
-            frappe.throw("You have already been rejected from this localhost.")
+            frappe.throw(
+                "You have already been rejected from this localhost.", frappe.PermissionError
```

Comment:
extremely nitpicky comment - rejection sounds a bit harsh. could we go with something like "localhost unable to accommodate you" or something along those lines maybe?

Reply:
too vague, this case is specific when a person is trying to re-apply to a localhost from which they were "Rejected". Makes error handling better imo 😬